src/main/java/tc/oc/pgm/maptag/StandardMapTags.java

```diff
@@ -0,0 +1,105 @@
+ */
+
+package tc.oc.pgm.maptag;
+
+import static com.google.common.base.Preconditions.*;
+
+import java.util.function.Predicate;
+import tc.oc.pgm.blitz.BlitzModule;
+import tc.oc.pgm.classes.ClassModule;
+import tc.oc.pgm.controlpoint.ControlPointModule;
+import tc.oc.pgm.core.CoreModule;
+import tc.oc.pgm.crafting.CraftingModule;
+import tc.oc.pgm.destroyable.DestroyableModule;
+import tc.oc.pgm.ffa.FreeForAllModule;
+import tc.oc.pgm.flag.FlagModule;
+import tc.oc.pgm.hunger.HungerModule;
+import tc.oc.pgm.map.MapModule;
+import tc.oc.pgm.map.PGMMap;
+import tc.oc.pgm.modules.InfoModule;
+import tc.oc.pgm.modules.InternalModule;
+import tc.oc.pgm.modules.LaneModule;
+import tc.oc.pgm.rage.RageModule;
+import tc.oc.pgm.score.ScoreModule;
+import tc.oc.pgm.teams.TeamModule;
+import tc.oc.pgm.terrain.TerrainModule;
+import tc.oc.pgm.timelimit.TimeLimitModule;
+import tc.oc.pgm.tnt.TNTModule;
+import tc.oc.pgm.wool.WoolModule;
+import tc.oc.pgm.worldborder.WorldBorderModule;
+
+public interface StandardMapTags {
+  StandardMapTag _4TEAMS = create("4teams", TeamModule.class, team -> team.getTeams().size() == 4);
+  StandardMapTag BLITZ = create("blitz", BlitzModule.class, blitz -> !blitz.isDisabled(null));
+  StandardMapTag CLASSES = create("classes", ClassModule.class);
+  StandardMapTag CORE = create("core", CoreModule.class);
+  StandardMapTag CP = create("cp", ControlPointModule.class);
+  StandardMapTag CRAFTING =
+      create("crafting", CraftingModule.class, crafting -> !crafting.getCustomRecipes().isEmpty());
+  StandardMapTag DESTROYABLE = create("destroyable", DestroyableModule.class);
+  StandardMapTag EVENTEAMS =
+      create("eventeams", TeamModule.class, team -> team.shouldRequireEven().orElse(false));
+  StandardMapTag FFA = create("ffa", FreeForAllModule.class);
+  StandardMapTag FLAG = create("flag", FlagModule.class);
+  StandardMapTag FRIENDLYFIRE =
+      create("friendlyfire", InfoModule.class, info -> info.getMapInfo().friendlyFire);
+  StandardMapTag INTERNAL = create("internal", InternalModule.class);
+  StandardMapTag INSTANTTNT =
+      create("instanttnt", TNTModule.class, tnt -> tnt.getProperties().instantIgnite);
+  StandardMapTag NO_HUNGER = create("nohunger", HungerModule.class);
+  StandardMapTag RAGE = create("rage", RageModule.class);
+  StandardMapTag RFW = create("rfw", LaneModule.class);
+  StandardMapTag SCOREBOX =
+      create("scorebox", ScoreModule.class, score -> !score.getScoreBoxFactories().isEmpty());
+  StandardMapTag TEAMS = create("teams", TeamModule.class);
+  StandardMapTag TDM =
+      create(
+          "tdm",
+          ScoreModule.class,
+          score -> score.getConfig().deathScore != 0 || score.getConfig().killScore != 0);
+  StandardMapTag TIMELIMIT =
+      create("timelimit", TimeLimitModule.class, timeLimit -> timeLimit.getTimeLimit().isPresent());
+  StandardMapTag VANILLAWORLD =
+      create("vanillaworld", TerrainModule.class, terrain -> terrain.getOptions().vanilla);
```

Comment:
Maybe call this `#worldgen` ?

Reply:
Isn't `vanillaworld` more user friendly?

![image](https://user-images.githubusercontent.com/4578571/71642830-136e0680-2cb2-11ea-9ab1-4dec64fcd74c.png)

Reply:
i'm not sure, if i didn't know all the tags and i had to guess, i don't think i would've ever guessed `vanillaworld` to find a map that had vanilla world generation

Reply:
Actually none of maps in [maps.oc.tc](https://github.com/OvercastNetwork/maps.oc.tc) repo uses this feature. I'll make it longer and call it `#vanillaworldgen`.